### PR TITLE
STYL: testUtils: Improve test outputs

### PR DESCRIPTION
**Pull request by jcfr**
* Update feature name in testing output to reflect the feature class.

* write "N/A" string when no data can be found

* write csv as text file using `w` instead of `wb` (this should help with
  the transition to py3)

### CODE REVIEW

File: tests/testUtils.py [hunk 1]
```diff
@@ -244,18 +245,18 @@ def writeCSV(self, data, fileName):
 
     {'id1' : {'f1':n1, 'f2':n2}, 'id2' : {'f1':n3, 'f2':n4}}
     """
-    csvFile = open(fileName, 'wb')
+    csvFile = open(fileName, 'w')
     csvFileWriter = csv.writer(csvFile)
     # get the headers from the first row
-    header = sorted(data[data.keys()[0]].keys())
+    header = list(data[list(data.keys())[0]].keys())
```
Comment (jcfr) [inline tests/testUtils.py]: @JoostJM This change was grabbed from 4f380d6 in #194 , is this expected ? I guess it simply create the CSV column in the order of the keys.

Comment (JoostJM) [inline tests/testUtils.py]: @jcfr I changed this as I forgot to add the feature class to the output (meaning the features from different classes ended up mixed together due to the sorting). However, the output now again contains also the feature classes, meaning applying the sorted() function nicely sorts the features alphabetically per featureclass (as feature classes are also sorted alphabetically). So in this case, I'd propose to revert this to `sorted(data[list(data.keys())[0]].keys())`

Comment (JoostJM) [inline tests/testUtils.py]: @jcfr, I'm rewriting this a bit, as it is a bit buggy if you e.g. only test the 'breast1' test case. In that case, no results are written as the first test case ('brain1') does not contain any calculated features. The test cases are based on the baseline. I'll rewrite it so, that only the test cases that have been actually tested are written to the result and diff files, with no files written if no tests were actually run.

Comment (jcfr) [inline tests/testUtils.py]: Makes sense. Glad I asked :smile: 

Here is a PR you could further improve. See https://github.com/Radiomics/pyradiomics/pull/197

**Pull request merged**